client/asset/btc/btc.go

```diff
@@ -846,7 +865,15 @@ func (btc *ExchangeWallet) split(value uint64, lots uint64, outputs []*output, i
 		return nil, false, fmt.Errorf("error creating change address: %w", err)
 	}
 
-	feeRate := btc.feeRateWithFallback(1) // these must fund swaps, so don't under-pay (is this an issue with no fundconf requirement?)
+	// This must fund swaps, so don't under-pay. TODO: get and use a fee rate
+	// from server, and have server check fee rate on unconf funding coins.
+	feeRate, err := btc.feeRate(1)
+	if err != nil {
+		// fallback fee rate is NO good here.
+		// feeRate = nfo.MaxFeeRate // maybe
+		return nil, false, fmt.Errorf("Unable to get optimal fee rate for pre-split transaction: %v", err)
+	}
+	feeRate += feeRate / 10 // cost of chaining unconf txn, don't gamble.
```

Comment:
If I'm not mistaken, we've been running with zero buffer and have yet to hit an issue. If a buffer gives us some comfort, so be it, but a 10% bump is a lot more than the couple of sats we should need.

Reply:
Well, I'm not so sure actually we haven't been bit, primarily because the estimate is the client's rather than server's.  This is a bad band-aid though.  I'll remove this little bump and we can focus on solving this properly.